proto/src/determined/agent/v1/agent.proto

```diff
@@ -49,7 +49,7 @@ message Agent {
   // A map of container id to all containers assigned to this agent.
   map<string, determined.container.v1.Container> containers = 4;
   // This field has been deprecated and will be empty.
-  string label = 5;
+  string label = 5 [deprecated = true];
```

Comment:
can we simply remove this and other protobuf field, instead of making them deprecated?

Reply:
If we do, `make -C proto check` fails.

for example,
```
buf lint
buf breaking --against buf.image.bin
src/determined/api/v1/agent.proto:12:1:Previously present field "5" with name "label" on message "GetAgentsRequest" was deleted.
```

Reply:
there's `make gen-buf-image` in there which'll force-regenerate the buf image. this is a protective measure against breaking an API accidentally but here we break it explicitly

Reply:
Thanks!